solution.cpp

```diff
@@ -1,23 +1,101 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
+ * Copyright &copy; 2015 Randy Tanizawa <rktanizawa@gmail.com>
  * Released under the [MIT License] (http://opensource.org/licenses/MIT)
  * ------------------------------------------------------------------------- */
 
 /**
- * TODO: program description
+ * Arabic numeral to Roman numeral conversion program
  */
 
-// TODO: `#include`s and function prototypes
+#include <iostream>
+#include <string>
+
+using std::cout;
+using std::cin;
+using std::string;
+
+int input;
+string answer;
+void Int_to_Roman(int input)
 
 // ----------------------------------------------------------------------------
 
-int main() {
-    // TODO
+void Int_to_Roman(int input)
+{
+    cout << " ----> ";
+	int Mtimes = (input / 1000);
+	switch (Mtimes){
```

Comment:
could switch directly on `input / 1000` rather than storing it in a variable first.